examples/index.html

```diff
@@ -16,13 +16,17 @@
 
 <nav class="scroll-section">
     <a href="/">
-        <img src="images/itowns_logo.png" width="245px"/>
+        <img src="images/itowns_logo.png" width="245px" id="itowns-logo">
```

Comment:
missing `/` at the end?

Reply:
`/` at the end of self-closing tags (such as `img`) is not needed in html5, as I red [here](https://stackoverflow.com/a/15149657)